db/migrations/0003_add_frames_table.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE frames (
```

Comment:
Add previous and next frame hashes. Also probably need to store the hash for this frame.